test/numbers.jl

```diff
@@ -2864,3 +2864,19 @@ let types = (Base.BitInteger_types..., BigInt, Bool)
 end
 
 @test !isempty(complex(1,2))
+
+@testset "iszero" begin
+    # Numeric scalars
+    for T in Iterators.flatten(subtypes.([AbstractFloat, Signed, Unsigned]))
```

Comment:
these types should probably just be written out - there's a type `SOE.Sgnd` in the core test that causes a failure here if it runs in the same worker